src/Symfony/Component/AssetMapper/MappedAsset.php

```diff
@@ -52,6 +52,7 @@ public function __construct(
         private array $dependencies = [],
         private array $fileDependencies = [],
         private array $javaScriptImports = [],
+        public ?string $integrity = null,
```

Comment:
If possible i would make it readonly as the digest ere.. so not a ppp

Reply:
Made it `readonly`, but why not a ppp?

Reply:
Yep you're right, it's in fact the exact opposite situation that generated problems, when we after we added no porperties on one of the cached class... many people had problems during upgrade.

😶

Reply:
Not sure to understand, but if old `MappedAsset`s are serialized in a cache, accessing their new `$integrity` property would crash; is that what we must be guarding from?

Reply:
Yes that was that, proof i'm not clear in my head about this. 

But yeah, should be attnetive that really was cause of frustrations for users.